README.md

```diff
@@ -141,6 +141,33 @@ Digest: <encryption verification>
 Status: Downloaded pega-docker.downloads.pega.com/platform/pega:<version>
 ```
 
+For details, see the examples listed in the preparation runbook:
+
+* [Preparing your local Linux system](docs/prepping-local-system-runbook-linux.md)
+* [Preparing your local Windows 10 system](docs/prepping-local-system-runbook-windows.md)
+
+Pegasystems uses a standard naming practice of hostname/product/image:tag. Pega images are available from the host site, pega-docker.downloads.pega.com. Pega maintains four types of required Docker images for Client-managed Cloud deployments of Pega Platform:
+
+ Name                                           | Description                                           | Tags     |
+------------------------------------------------|-------------------------------------------------------|----------|
+`platform/installer`                            | A utility image with which you install all of the Pega-specific rules and database tables in the “Pega” database that you have configured for your deployment. This installation is required before a deployment can take place.| `<version>` |
+`platform/pega`                                 | (Download required) Deploys Pega Platform with its customized version of the Tomcat application server.| `<version>` or `<version>-YYYYMMDD` |
+`platform/search`                               | (Download required) Deploys the required search engine for Pega Platform search and reporting capabilities. This Docker image contains Elasticsearch and includes all required plugins.| `<version>` or `<version>-YYYYMMDD`. |
+`platform-services/search-n-reporting-service`  | An image that deploys the required search engine for the latest generation of search and reporting capabilities in Pega Infinity as a backing service. This Docker image contains Pega Search and Reporting Services. For more information about this service, see [Instructions to configure the Pega backingservices](charts/backingservices/README.md). | `<version>` |
+`constellationui/service`  | An image that deploys the constellationui service for the latest generation of react based ui. For more information about this service, see [Instructions to configure the constellationui service](charts/pega/charts/constellation/README.md). | `<version>` |
+
+
+For the `platform/installer` image, the :tag represents the version of Pega you want to install, for example the tag :8.5.1 will install Pega Platform version 8.5.1.
```

Comment:
I'd recommend modifying these details to be closer to 8.8 and October 2022 so it doesn't look so stale>>

For the `platform/installer` image, the :tag represents the version of Pega you want to install, for example the tag :8.8.0 will install Pega Platform version 8.8.0 (the GA version).
For `platform/pega` and `platform/search` images, Pega also offers an image with a version tag appended with a datestamp using the pattern `pegaVersion-YYYYMMDD` to indicate the version and the date that Pega built the image. For example, if you pull the `platform/pega` with a tag, `pega:8.5.1-20221018`, the tag indicates that Pega built this 8.8.0 image on 18 October 2022. Using the version tag without the datestamp will always point to the most recently built image for that version.